lib/livebook/notebook/explore/distributed_portals_with_elixir.livemd

```diff
@@ -705,6 +705,12 @@ IO.puts Node.get_cookie()
 
 Now paste the result of the other node name and its cookie in the inputs below:
 
+```elixir
```

Comment:
I wasn't sure if there should be text here explaining why the Kino install is needed or if it's self-explanatory enough. I chose not to include it because I didn't want to distract readers by having to explain why this is needed since Kino is not the focus of this notebook. But, I'm open to any suggestions.